processing/src/main/java/org/apache/carbondata/processing/loading/CarbonDataLoadConfiguration.java

```diff
@@ -348,6 +356,33 @@ public void setCardinalityFinder(DictionaryCardinalityFinder cardinalityFinder)
     return complexKeyGenerators;
   }
 
+  public Boolean[][] createComplexDictionaryFieldIdentification() {
```

Comment:
If not required, please remove this method

Reply:
removed

Reply:
Removed